contracts/kleros/BosonKlerosConnector.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8;
+// THIS CONTRACT IS IMPLEMENTED FOR INOFRMATION PURPOSES AND SHOULD NOT BE USED IN PRODUCTION
+
+import { IArbitrator } from "./IArbitrator.sol";
+import { IArbitrable } from "./IArbitrable.sol";
+import { IMetaEvidence } from "./IMetaEvidence.sol";
+import { IEscalatable } from "../interfaces/escalation/IEscalatable.sol";
+import { IEscalationResolver } from "../interfaces/escalation/IEscalationResolver.sol";
+
+contract BosonKlerosConnector is IEscalatable, IArbitrable, IMetaEvidence {
```

Comment:
This should be organized under `protocol/clients/kleros/BosonKlerosConnector`. It also probably needs to be upgradeable.